docs/src/pages.js

```diff
@@ -68,4 +73,27 @@ datagrid.children =
         { pathname: '/components/data-grid/group-pivot', title: '🚧 Group & Pivot' },
       ];
 
+dataGridComponentAPI.children =
+  process.env.PULL_REQUEST === 'false'
```

Comment:
I doubt that we should approach unstable docs page like this. We duplicate the content. Instead, we could make it additive. For now, maybe remove this branch for now?

Reply:
I did what was done for the `components` part (above in the same file)
But I do not understand the goal of this ternary to be honest.

Reply:
The goal was to be able to work on new pages in dev mode without releasing them in production.

Reply:
Is the process of releasing a new doc page documented ?
I suppose that at some point we need to add it to the core repo for it to be released.

Reply:
> Is the process of releasing a new doc page documented ?

It's not documented when it comes to X. For the core, it's inside the CONTIRBUTION.md guide. Understanding how it works requires understanding how the production is structured. To make it short, we have two different websites (core, x). In production, we use a CDN, we proxies this two webistes to share the same URLs. 

So the sync is about:

1. making sure we share the same code infra for consistency (e.g. same CSS). This is done with x importing the core repository (monorepo dependency pointing to a git commit) and using the code of the docs of the core.
2. make sure the links in X are up to date. this is currently done automatically by importing the pages from the core repo. In the future, we will solve that by not showing the links in the first place. At least, it's what we have discussed doing with @danilo-leal.
3. make sure the links in core are up to date, pointing to X correctly. This is done manually, by the X team making PRs on the core. In the future, we will solve that by not showing the links.

Reply:
OK it's a lot clearer now.
Indeed the 3. is a limitation that would be nice to remove but I suppose it's far from being simple.